src/data/customer/Job.js

```diff
@@ -275,3 +275,26 @@ export const createJobsAsync = async (routeIds, token) => {
     return apiResponseErrorHandler(e);
   }
 };
+
+/**
+ * delete Jobs
+ * @param {string} jobIds comma separated string ex: 1234,3455
+ * @param {string} token
+ * @return {object} Promise resolve/reject
+ */
+export const removeJobsAsync = async (jobIds, token) => {
+  try {
+    await axios({
+      method: 'DELETE',
+      // TODO: need to update this endpoint again when API starts accepting multiple job ids
+      url: endpoints.API_V3.JOB.replace('{0}', jobIds),
```

Comment:
@vinay3206 will this be any different if the API changes? You would still pass 'jobIds' no?

Reply:
@maartenhemmes  as discussed with nin. It will be probably changed to query string params

Reply:
ok, I think they will still fix it within this sprint, so we might as well wait for that? what do you think?